app/views/assignment/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Kitchen Schedule</h1>
+
+<table>
+
+<% @weeks.each do |w| %>
+  <tr>
+    <td>
+      <%= "#{w.starts_at.strftime('%B %d, %Y')}" %>
```

Comment:
I might use a presenter method here that overrides `starts_at` to use `week.starts_at` in the view for the same behaviour. In the helper I'd use the `l` method (short for `localize`) that translates timestamps into a locale-specific format. In this case, you could improve this without a presenter by using the `l` helper directly here or by creating a helper method (maybe inside `app/helpers/timestamp_helpers.rb`) so you could do `readable_date(week.starts_at)`. Finally, I'd use `'%B %e, %Y'` because I think leading zeroes look funny in most contexts.

This is an optional change.